README.md

```diff
@@ -1 +1,4 @@
+[![Build Status](https://travis-ci.org/andela/ah-frontend-summer.svg?branch=develop)](https://travis-ci.org/andela/ah-frontend-summer)
```

Comment:
Is this change part of the story?
You can revert.

Reply:
@collinewait Yes let me do that

Reply:
Resolve conflicts

Reply:
@collinewait resolved conflicts